Formula/c/cloudflare-quiche.rb

```diff
@@ -23,10 +23,21 @@ class CloudflareQuiche < Formula
   def install
     system "cargo", "install", *std_cargo_args(path: "apps")
 
-    system "cargo", "build", "--lib", "--features", "ffi,pkg-config-meta", "--release"
-    lib.install "target/release/#{shared_library("libquiche")}"
+    system "cargo", "build", "-j", "1", "--lib", "--features", "ffi,pkg-config-meta", "--release"
```

Comment:
why this change?

Reply:
oh this was only for debugging. need to be removed.